utils/general.py

```diff
@@ -53,6 +54,25 @@ def download_file_blob_to_dir(directory, file_blob):
     os.remove(file_destination)
 
 
+def move_downloaded_data_to_single_directory(directories):
+
+    for directory in directories:
+        if not os.path.isdir(directory + "train"):
+            os.makedirs(directory + "train")
+        if not os.path.isdir(directory + "val"):
+            os.makedirs(directory + "val")
+
+        folders = os.listdir(directory)
+        for folder in folders:
+            if folder != "train" and folder != "val":
+                if folder.startswith("train"):
```

Comment:
Can I get you to add some comments here? It seems very strange to filter out "train" only to check if it starts with "train" on the very next line.

Reply:
Fixed c11a525c00152290ce52eea5482ba430a485990b